docs/user/build-customization.rst

```diff
@@ -373,19 +373,15 @@ Take a look at the following example:
      os: "ubuntu-22.04"
      tools:
        python: "3.10"
-     jobs:
-       post_create_environment:
-         # Install uv
-         - pip install uv
-       post_install:
-         # Install dependencies with 'docs' dependency group
-         # VIRTUAL_ENV needs to be set manually for now.
-         # See https://github.com/readthedocs/readthedocs.org/pull/11152/
-         - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH uv pip install .[docs]
-
-   sphinx:
-     configuration: docs/conf.py
-
+     commands:
+       - asdf plugin add uv
+       - asdf install uv latest
+       - asdf global uv latest
+       - uv venv
+       - uv pip install .[docs]
+       - .venv/bin/python -m sphinx -T -b html -d docs/_build/doctrees -D language=en docs $READTHEDOCS_OUTPUT/html
+
+You can use ``-r docs/requirements.txt``, etc. instead as needed. And mkdocs would use ``.venv/bin/python -m mkdocs build --site-dir $READTHEDOCS_OUTPUT/html``.
```

Comment:
```suggestion
You can use ``-r docs/requirements.txt``, etc. instead as needed. And mkdocs would use ``. .venv/bin/activate && NO_COLOR=1 mkdocs build --strict --site-dir $READTHEDOCS_OUTPUT/html``.
```

For some reason, MkDocs-macros doesn't work if you just run it from the venv directly. Will look into why that happens, for now, this is probably a good idea to avoid mistakes.

Reply:
```suggestion
You can use ``-r docs/requirements.txt``, etc. instead as needed. And mkdocs would use ``NO_COLOR=1 .venv/bin/mkdocs build --strict --site-dir $READTHEDOCS_OUTPUT/html``.
```

I was told it should work, so this should be fine.

Reply:
Yes, it was a custom plugin. This is fine.